app/views/_user_util_links.html.erb

```diff
@@ -9,14 +9,13 @@
       <% end %>
       <a class="btn btn-default dropdown-toggle" data-toggle="dropdown" href="#"><span class="sr-only">select for additional menu options</span><span class="caret" ></span></a>
       <ul class="dropdown-menu dropdown-menu-right" role="menu">
-        <% if can? :create, GenericFile %>
-          <li><%= link_to raw('<span class="glyphicon glyphicon-upload"></span> upload'), sufia.new_generic_file_path, id: 'upload_nav_link' %></li>
-        <% end %>
         <li><%= link_to raw('<span class="glyphicon glyphicon-dashboard"></span> my dashboard'), sufia.dashboard_index_path, id: 'dashboard_nav_link' %></li>
         <li><%= link_to raw('<span class="glyphicon glyphicon-user"></span> my profile'), sufia.profile_path(current_user), id: 'profile_nav_link' %></li>
-        <%= render partial: 'users/user_util_links_extra' %>
-        <% if can? :manage, 'anything' %>
-          <li><%= link_to raw('<span class="glyphicon glyphicon-king"></span> administration'), role_management.roles_path, id: 'logout_nav_link' %></li>
+        <% if current_user.sunset_editor? %>
```

Comment:
I don't like that this is what we have to do. It's pretty clearly a hacky solution, but this is a remarkably hacky app. Ideally this would be something handled on a controller, but it's not realistic for the timeline we have and what has to be done.